tests/modules/lib/cddl-gen/encode/test.cddl

```diff
@@ -0,0 +1,13 @@
+;
+; Copyright (c) 2021 Nordic Semiconductor ASA
+;
+; SPDX-License-Identifier: Apache-2.0
+;
+
+
+Timestamp = bstr .size 8
```

Comment:
Is this file is identical with the one used for `raw_encode` should we pull it to a parent directory and have only one copy instead of duplicating them?

Reply:
Yep, it's the same for decode and encode. It's unneeded for raw_encode actually, but I moved it to tests/modules/lib/cddl-gen/cddl/test.cddl so it's not duplicated.